AWS High Availability Architecture/README.md

```diff
@@ -0,0 +1,232 @@
+
+A highly available architecture involves multiple components working together to ensure uninterrupted service during a specific period. This also includes the response time to users’ requests. Namely, available systems have to be not only online, but also responsive.
+
+Implementing a cloud computing architecture that enables this is key to ensuring the continuous operation of critical applications and services. They stay online and responsive even when various component failures occur or when a system is under high stress.
+
+Highly available systems include the capability to recover from unexpected events in the shortest time possible. By moving the processes to backup components, these systems minimize downtime or eliminate it. This usually requires constant maintenance, monitoring, and initial in-depth tests to confirm that there are no weak points.
+
+**Application:** If you want to deploy a website and want your website to be fast, secure and available all the time even in the time of some failure in the network then`High Availabilty Architecture`comes in play.
+
+It is important to create `High Availability Architecture`to work efficiently without any delay due to failure.
+
+In this MicroByte we will perform following activities:
+
+- We will Launch an instance using AWS CLI
+- We will configure Webserver 
+- We will Create an EBS Volume
+- We will Attach EBS Volume
+- We will Create Partition
+- We will create S3 Bucket
+- We will Create Cloudfront Distribution
+
+I hope you will learn by doing the Activities and will perform Task given in the Activities #LearnByDoing
+
+# Prerequisites
+
+- You should have AWS account, you can create [AWS free tier account](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc).
+- Install AWS CLI in [Windows](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-windows.html),[Mac](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-mac.html) or in [Linux](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html).
+- Configure [AWS CLI with IAM user](https://bauddhik-geek.blogspot.com/2020/10/working-in-aws-using-aws-cli-with-help.html).
+- It is preferred that you are aware [HTML basics](https://learn.crio.do/home/me/ME_HTML_CSS) and [Linux](https://learn.crio.do/home/me/ME_LINUX1)
+
+The Activities that we are going to perform on AWS are free in `AWS free tier plan`but you should know the limitation of free tier plan so, go to the [link](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc) and check limitation.
+
+### Activity 1: Launch an Instance using AWS CLI
+First configure the AWS CLI in your [window cmd](https://www.online-tech-tips.com/computer-tips/how-to-use-dos-command-prompt/) by providing Access Key, Secret Key and region name.

```

Comment:
A learner might be using Linux/Mac-OS so would have a terminal. It seems that is instruction will be similar for terminal too, kindly write it that way. Writing only `window cmd` might be a little misleading, plus it's `Windows`.

Reply:
U can mention that the environment u have used is windows and so some things maybe be different for different OS